nl/nl_linux.go

```diff
@@ -46,32 +46,30 @@ func GetIPFamily(ip net.IP) int {
 	return FAMILY_V6
 }
 
-var nativeEndian binary.ByteOrder
+var nativeEndian = func() binary.ByteOrder {
```

Comment:
I'd add a comment, stating something along the lines of "native byte order as determined at program startup" and "assuming it doesn't change at runtime".